runtime/j9vm/javanextvmi.cpp

```diff
@@ -428,9 +428,6 @@ JVM_VirtualThreadUnmountEnd(JNIEnv *env, jobject thread, jboolean lastUnmount)
 
 	if (lastUnmount) {
 		vmFuncs->freeTLS(currentThread, threadObj);
```

Comment:
curious if those could be moved too (even if could be called within freeContinuation)?